src/modules/blocks/tickets/template.js

```diff
@@ -18,13 +18,14 @@ class Tickets extends PureComponent {
 		isSelected: PropTypes.bool,
 		isSettingsOpen: PropTypes.bool,
 		clientId: PropTypes.string,
+		header: PropTypes.string,
```

Comment:
this is actually not what i meant in my previous comment, as we don't use the `header` prop in the template, I don't think it makes sense to have this here.